test/types/main.ts

```diff
@@ -1,9 +1,12 @@
 import { getDocument } from "pdfjs-dist";
+import { EventBus } from "pdfjs-dist/web/pdf_viewer.component";
```

Comment:
The contents of the `typestest` folder is somewhat different from `pdfjs-dist`. Notably, it's missing a `legacy` folder so it's not possible to check that the `external/dist/legacy/web/pdf_viewer.d.ts` file is doing the right thing during CI. Could be good to do in a future change!

Reply:
The TypeScript definitions provided can be seen as a best-effort solution, hence we don't want to *further* increase test run-time for `typestest` by testing e.g. the `legacy`-folder (those builds are only provided for older environments anyway).